### PR TITLE
[script] install shfmt

### PR DESCRIPTION
**Pull request by bukepo**
This commit installs shfmt in script/bootstrap.

### CODE REVIEW

**Comment by simonlingoogle** [comment]
> This commit installs shfmt in script/bootstrap.

why not just install it in `make-pretty`?

File: script/bootstrap [hunk 1]
```diff
@@ -58,6 +58,9 @@ install_packages_apt()
 
     # add mdv for local size report
     python3 -m pip install mdv || echo 'Failed to install markdown render. Install it manually if you need.'
+
+    # add shfmt for shell pretty
+    command -v shfmt || brew install shfmt || (sudo apt-get install --no-install-recommends -y snapd && sudo snap install shfmt)
```
Comment (jwhui) [inline script/bootstrap:63]: Does attempting to use `brew install` make sense in `install_packages_apt()`?

Comment (bukepo) [reply script/bootstrap:63]: Yes, snap doesn't support home dir not being at /home. Linuxbrew works perfect. So try brew first.

**Comment by bukepo** [comment]
> > This commit installs shfmt in script/bootstrap.
> 
> 
> 
> why not just install it in `make-pretty`?

I don't see other style tools are installed in that script.

File: script/bootstrap (hunk 1 above)
Comment (jwhui) [reply script/bootstrap:63]: Makes sense. Can we add a comment since having `brew install` inside `install_packages_apt()` is not intuitive?

Comment (bukepo) [reply script/bootstrap]: added.

**Comment by bukepo** [comment]
There are no logs of the docker build failures. And I cannot re-trigger the build. I will close and re-open and see if docker build is rebuilt and hopefully I can get the error logs.

**Comment by bukepo** [comment]
close and reopen to trigger docker hub build.

**Pull request merged**